packages/iTwinUI-react/src/core/Modal/Modal.tsx

```diff
@@ -62,16 +47,12 @@ export type ModalProps = {
    * @default document
    */
   ownerDocument?: Document;
-  /**
-   * Type of the modal.
-   * @default 'default'
-   */
-  styleType?: 'default' | 'fullPage';
   /**
    * Content of the modal.
    */
   children: React.ReactNode;
-} & Omit<CommonProps, 'title'>;
+} & Pick<DialogProps, 'isOpen' | 'styleType'> &
+  Omit<CommonProps, 'title'>;
 
 /**
  * Modal component which can wrap any content.
```

Comment:
Why did modal test images change?

Reply:
No idea, tried to move imgs from main branch but then tests failed.